dls-controls/malcolmjs
# Feature/expander #41

Changes:
- Hooked up the default expanded state of the groups
- Added streaming changes to the test server to prove the group doesn't open and close when the state changes.
- Updated the attribute name to use the metadata label
- Filter anything with the tag widget:icon rather than the name icon
- Generally updated the group/attribute container/led widget so it looks right in the attribute details panel.
- Fixed snapshot tests so they don't contain all the theming variables (only for the snapshots in this PR, these will be addressed as we go forward).
- Moved the LED widget into the malcolmWidgets folder

Still to do:
- Add description as a tooltip

To test:
- Run `npm start` and navigate to http://localhost:3000/gui/?block=PANDA:SEQ1 and observe the led attributes in the outputs section. 

connect to #41
